__tests__/tooling/index.test.js

```diff
@@ -82,6 +82,69 @@ describe('#url([selected])', () => {
         'https://subdomain.example.com:8080/v3'
       );
     });
+
+    it('should use defaults', () => {
+      expect(
+        new Oas({
+          servers: [{ url: 'https://example.com/{path}', variables: { path: { default: 'path' } } }],
```

Comment:
do we handle the case for when we have a variable-url but no actual variables—I only ask because I remember we was that recently (wouldn't have figured it was valid otherwise)

Reply:
So this?

```
servers: [
  { url: 'https://{region}.node.example.com/v14' }
}
```

I can add some test cases.

Reply:
yes!

Reply:
Looks like this PR fixes that up as well.